contracts/axelar-gateway/tests/testdata/call_contract.golden

```diff
@@ -1,3 +1,9 @@
-contract: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
```

Comment:
Since the event topics/data methods are now auto-generated rather than custom, we can format the struct itself